tests/containers/kubectl.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test the kubectl utility
+# - List pods
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+use version_utils;
+use publiccloud::utils;
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
+use containers::k8s;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    # kubectl is in the container module (on SLES)
+    add_suseconnect_product(get_addon_fullname('contm')) if (is_sle);
+    zypper_call("-n in kubernetes-client");
+
+    # Record kubectl version and check if the tool itself is healthy
+    record_info("kubectl", script_output("kubectl version --client --output=json"));
```

Comment:
If we are going to use the install_kubectl will be interesting to move this line inside the k8s library after installing to have this benefit in all tests that are using this.

Reply:
I think this output is only interesting in the test run where we test `kubectl`. In other test cases this spams the webui with unnecessary `record_info` boxes

Reply:
ok